app/views/requests/_form.html.erb

```diff
@@ -35,32 +35,32 @@
   </div>
 
   <div class="field form-group">
-    <%= form.label :cpu_cores, "CPU-Cores" %>
+    <%= form.label :cpu_cores, "CPU cores" %>
     <%= form.number_field :cpu_cores, min: 0, class: 'form-control', id: 'cpu' %>
   </div>
 
   <div class="field form-group">
-    <%= form.label :ram_mb, "RAM in MB" %>
+    <%= form.label :ram_mb, "RAM in GB" %>
     <%= form.number_field :ram_mb, min: 0, class: 'form-control', id: 'ram' %>
   </div>
 
-  <div class="field">
-    <%= form.label :storage_mb, "Storage in MB" %>
+  <div class="field form-group">
+    <%= form.label :storage_mb, "Storage in GB" %>
```

Comment:
Are you sure this works?
If the request already has a storage size set, will it not display the value in MB here?